switchtheme.module

```diff
@@ -187,6 +201,34 @@ function switchtheme_block_view($delta = '') {
   }
 }
 
+/**
+ * returns an unordered list of active themes to switch to.
+ */
+
+function switchtheme_return_itemlist_block() {
+  $themes = list_themes();
+  $enabledThemes = []; 
+
+  $current_url = url(current_path(), array('absolute' => TRUE));
+  foreach ($themes as $key => $theme) {
+    if ($theme->status) { // if enabled
+      $items[] = array(
+        'data' => l($theme->name, $current_url . '?theme=' . $theme->name),
+      );
+    } 
+  }
+
+  $content = [
+    '#theme' => 'item_list',
+    '#list_type' => 'ul',
+    '#items' => $items,
+    '#attributes' => ['class' => 'mylist'],
+    '#wrapper_attributes' => ['class' => 'container'],
+  ]; 
```

Comment:
You are defining the `$enabled_themes` array variable earlier (well, `$enabledThemes` but that is not acceptable according to our PHP coding standards), but you never use it. I am assuming that it's meant to be used here 🙂 

```suggestion
      $enabled_themes[] = array(
        'data' => l($theme->name, $current_url . '?theme=' . $theme->name),
      );
    } 
  }

  $content = [
    '#theme' => 'item_list',
    '#list_type' => 'ul',
    '#items' => $enabled_themes,
    '#attributes' => ['class' => 'mylist'],
    '#wrapper_attributes' => ['class' => 'container'],
  ]; 
```

Reply:
...finally, that `mylist` class is not used anywhere. It should either be removed, or changed to something useful, like `switchthem_themes_list` or similar.